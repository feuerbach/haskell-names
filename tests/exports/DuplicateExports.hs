module DuplicateExports (A(A), A) where

data A = A

