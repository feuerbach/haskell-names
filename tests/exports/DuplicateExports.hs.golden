[ Data
    { symbolModule = ModuleName () "DuplicateExports"
    , symbolName = Ident () "A"
    }
, Constructor
    { symbolModule = ModuleName () "DuplicateExports"
    , symbolName = Ident () "A"
    , typeName = Ident () "A"
    }
]